core/scene/view/cell.view.d.ts

```diff
@@ -12,6 +12,7 @@ export interface ICellModel {
 export declare class CellView {
 	constructor(model: ICellModel);
 
+	columnKey: any;
```

Comment:
revert changes u  gan get this throught the model

Reply:
fixed